istio/istio.go

```diff
@@ -627,6 +627,11 @@ func (iClient *Client) ApplyOperation(ctx context.Context, arReq *meshes.ApplyRu
 		return &meshes.ApplyRuleResponse{
 			OperationId: arReq.OperationId,
 		}, nil
+	case runVet:
+		err = iClient.runVet()
+		return &meshes.ApplyRuleResponse{
+			OperationId: arReq.OperationId,
+		}, err
```

Comment:
We are returning nil everywhere, is this fine?

Reply:
`err` will anyway be nil if no error, so should be.